### ISSUE TITLE
podman system df: --format: needs newline terminators

### ISSUE THREAD

**Issue by edsantiago** [containers/podman#6962]
podman-system-df(1) documents the `--format` flag as follows:
```
$ podman system df --format "{{.Type}}\t{{.Total}}"
Images          1
Containers      5
Local Volumes   1
```
Actually results:
```console
$ podman system df --format "{{.Type}}\t{{.Total}}"
Images\t4Containers\t0Local Volumes\t0%    <--- the "%" means "no trailing newline"
```

**Comment by vrothberg** [comment]
@ryanchpowell, is that something you'd like to tackle?

**Comment by ryanchpowell** [comment]
@vrothberg Yes this seems like a good job for me

**Comment by edsantiago** [comment]
This was fixed by #7156
